tests/ltp/patches/fix_mkdir_mkdir02.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/testcases/kernel/syscalls/mkdir/mkdir02.c b/testcases/kernel/syscalls/mkdir/mkdir02.c
```

Comment:
Summary description of patch is missing. See other patches.

Reply:
Done